### PR TITLE
Selecting references entry

### PR DESCRIPTION
**Pull request by B-Bonner**
DH entry about the 'selecting references' feature and content changes we made to make the process clearer to candidates.

(Copied an existing DH entry and then populated it with content for this entry)

### CODE REVIEW

File: app/posts/apply-for-teacher-training/2021-07-14-selecting-references.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+title: Selecting references
+description: Letting candidates choose which references to submit with their application
+date: 2021-07-14
```
Comment (paulrobertlloyd) [inline app/posts/apply-for-teacher-training/2021-07-14-selecting-references.md]: Use today’s date:

```suggestion
date: 2021-07-19
```

…and update the file name to match: `2021-07-19-selecting-references.md`

File: app/posts/apply-for-teacher-training/2021-07-14-selecting-references.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+Previously candidates were unable to choose which 2 references would be sent with their application to providers. They would request several references and whichever 2 they received first would be sent with their application.
+
+We decided to build a feature that allows candidates to select which 2 references they want to send to providers.
+
+{{ appScreenshots({
+  hideContents: true,
+  items: [{
+    text: "Select 2 references link on task list",
+    img: {
+      src: "Select-2-references-link.png"
```
Comment (paulrobertlloyd) [inline app/posts/apply-for-teacher-training/2021-07-14-selecting-references.md]: Use lowercase filenames for images (i.e. rename the image files, and update the references in this markdown file)

```suggestion
      src: "select-2-references-link.png"
```

**Review by paulrobertlloyd — CHANGES_REQUESTED** [review]
The images are not showing. This is because the path to the image folder needs to be the same as the slug of the markdown file. For example, if your Markdown file is saved here:

`app/posts/apply-for-teaher-training/2021-07-19-selecting-references.md`

The folder of images needs to be in a folder with the same name:

`app/images/apply-for-teacher-training/selecting-references/`

(We really should document this somewhere…)

File: app/posts/apply-for-teacher-training/2021-07-14-selecting-references.md (hunk 1 above)
Comment (B-Bonner) [reply app/posts/apply-for-teacher-training/2021-07-14-selecting-references.md]: Done.

File: app/posts/apply-for-teacher-training/2021-07-19-selecting-references.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+This was also a waste of referees’ time, because they were writing references that could not be used.
+
+In order to reduce the number of support queries, we decided to give candidates more information about what their referees will need to do. That way, they could make better decisions about who to choose as referees.
+
+We’ve now changed the guidance around choosing referees to include more information about what providing a reference will involve, as well as the word limit. We also explicitly say that references sent by email will not be accepted.
+
+{{ appScreenshots({
+  hideContents: true,
+  items: [{
+    text: "'Choose your referees' page",
```
Comment (paulrobertlloyd) [inline app/posts/apply-for-teacher-training/2021-07-19-selecting-references.md]:
```suggestion
    text: "‘Choose your referees’ page",
```

**Review by paulrobertlloyd — APPROVED** [review]

File: app/posts/apply-for-teacher-training/2021-07-19-selecting-references.md (hunk 3 above)
Comment (B-Bonner) [reply app/posts/apply-for-teacher-training/2021-07-19-selecting-references.md]: Updated.

**Comment by B-Bonner** [comment]
[Preview post](https://deploy-preview-397--bat-design-history.netlify.app/apply-for-teacher-training/selecting-references/)

**Comment by EmmaFrith** [comment]
@B-Bonner maybe we could say why we decided to do this, to give some context for the change

Screenshot 2021-07-19 at 17 40 53

**Comment by EmmaFrith** [comment]
It'd be good to have some context about why - although this might have been a 'just because' moment!
Screenshot 2021-07-19 at 17 54 58

**Comment by B-Bonner** [comment]
[Preview link](https://deploy-preview-397--bat-design-history.netlify.app/apply-for-teacher-training/selecting-references/)

**Pull request merged**